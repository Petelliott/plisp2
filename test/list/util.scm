(println (reverse '())
         (reverse '(1))
         (reverse '(1 2))
         (reverse '(1 2 3))
         (reverse '(1 2 3 4)))

(println (list (+ 1 2) 3 8)
         (list))

(println (length '())
         (length '(1))
         (length '(1 2 3 4 5 6)))
