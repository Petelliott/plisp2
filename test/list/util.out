() (1) (2 1) (3 2 1) (4 3 2 1)
(3 3 8) ()
0 1 6
